specs/integration/events/conversation_created.spec.ts

```diff
@@ -24,13 +27,26 @@ describe('Event conversation:create', () => {
   beforeEach(async () => {
     workerSpy = sinon.spy(webhookJob.worker, 'processJob')
     queueSpy = sinon.spy(webhookJob.queue, 'add')
+    
+    // setup webhook integration
+    storeWebhookIntegrationSecret(webhookIntegrationSecret)
```

Comment:
```suggestion
    await storeWebhookIntegrationSecret(webhookIntegrationSecret)
```

always await promises

Reply:
(this applies to other tests below of course)